app/views/layouts/mobile_preview.html.slim

```diff
@@ -0,0 +1,6 @@
+doctype html
+html
+  head
```

Comment:
I'm not sure how is this working. Don't we need to load the all the front facing assets (css, js)?

Reply:
This layout is a simple wrapper for the iframe, which loads the actually campaign page.

Reply:
I understand this is a simple wrapper. But by using this layout you don't use the regular layout for pages, the one that holds all assets for pages. That's whats confusing me

Reply:
Those load inside the iframe I assume.